src/hooks/requests/useTasks.ts

```diff
@@ -21,11 +22,17 @@ export const useFetchTasks = (initialFilters?: TasksPayloadType) => {
   } = useFilters<TasksPayloadType>(initialFilters)
 
   const { isLoading, isError, data, refetch } = useQuery<TasksResponse>({
-    queryKey: ['tasks', filters],
+    queryKey: ['tasks'],
```

Comment:
**[blocker]**

Won't work exactly like this, since we have the "Minu ülesanded" and "Ootel ülesanded" lists in the same view. Currently it will show the same data for both.